Tasks/google-play-release/GooglePlay.ts

```diff
@@ -777,22 +777,27 @@ function resolveGlobPaths(path: string): string[] {
 }
 
 /**
- * Get obb file for the apk from the parent directory.
- * @returns {string[]} path of the obb file if present else null
+ * Get obb file for the apk from the same directory as apk.
```

Comment:
This is a big breaking change for existing users. Is there a reason why we want to look in the same directory instead of keeping with the current setup of looking in the parent directory?

Reply:
So the reordering of folder structure in our pipeline is proving to be more costly rather than decreasing the effort. Moreover after some discussion, we thought it would make more sense for the expansion file to be present in the same folder as the apk(what do you think?). If it's possible to make any changes to not break the previous feature, I would be happy to get any suggestions and make the changes.

Reply:
Could it makes sense maybe to add some switch which allows to select behavior regarding strict obb file name format/path to obb file, and set it by default to behave like it was previously?

Reply:
Something like another check box in the UI? I was also wondering if we can by any chance know the number of users currently using the previous feature

Reply:
Hi team, any suggestions I can implement here? Or should we just make the changes and notify the existing users?

Reply:
No we can't break existing customers. Can you add an option, like @anatolybolshakov recommened, where you can pass the directory to use and have it default to the parent directory. Then for your instance you can override the default?